app/lib/airtable.server.ts

```diff
@@ -57,7 +57,7 @@ const airtableResponseSchema = <TSchema extends AnySchema>(
     records: z.array(recordSchema),
   });
 
-const stringArraySchema = z.array(z.string());
+const stringArraySchema = z.array(z.string().nullable());
```

Comment:
Why would Airtable be sending nulls in the array? I haven't observed this to be the case when interacting with the API, I think the nullable can be removed.

Reply:
It does for calculated fields on related columns
![image](https://user-images.githubusercontent.com/703800/211270890-d67098b5-05f7-4c98-ba84-5680e0dc4aba.png)